packages/nvidia_nat_memmachine/tests/test_memmachine_integration.py

```
@@ -0,0 +1,454 @@
+    """
+    base_url = os.getenv("MEMMACHINE_BASE_URL", "http://localhost:8080")
+    if not base_url.startswith("http"):
+        base_url = f"http://{base_url}"
+    
+    try:
+        # Try to import and use MemMachineClient to check server availability
+        from memmachine import MemMachineClient
+        
+        client = MemMachineClient(base_url=base_url, timeout=5.0)
+        client.health_check(timeout=5.0)
+        return base_url
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)